packages/framework/src/Controller/Admin/SelectAdminDomainsController.php

```diff
@@ -57,6 +58,10 @@ public function filterDomainsAction(Request $request): Response
             if (count($data) === 0) {
                 $this->addErrorFlash(t('Please select at least one domain.'));
             } else {
+                if (array_values($data) === $this->domain->getAllIds()) {
```

Comment:
Can we be sure the order will always be the same? 
Isn't comparing the count of items sufficient?